godel_robots/abb/abb_file_suite/src/ftp_upload.cpp

```diff
@@ -0,0 +1,139 @@
+#include "ftp_upload.h"
```

Comment:
Much of this code is copied from existing libCURL examples. Should include reference to that here.